Packs/QualysFIM/Integrations/Qualys/QualysFIM.py

```diff
@@ -204,7 +204,7 @@ def list_events_command(client: Client, args: dict):
     Returns:
         CommandResults: outputs, readable outputs and raw response for XSOAR.
     """
-    sort = json.dumps([{'dateTime': SORT_DICTIONARY.get(str(args.get('sort')))}])
+    sort = json.dumps([{'dateTime': SORT_DICTIONARY.get(str(args['sort']))}])
```

Comment:
This is not a required argument, are you sure that this code is safe?

Reply:
This is not required but it has default value so if the user doesn't use this argument the code will get the default value, so its safe